generate-persisted-query-manifest: support `addTypename: false` (#483)
apollographql/apollo-utils: packages/generate-persisted-query-manifest/README.md

  }),
};
```

- `addTypename`: `boolean`

Whether to add `__typename` fields to selection sets in operations. Defaults to true; you should set this to false if you also pass `addTypename: false` to your `InMemoryCache` constructor in your app. (This can also be helpful if you are using this tool on a codebase that does not actually use Apollo Client Web, though it is not designed for that purpose.)

> Note that the ability to pass `addTypename: false` will be removed in Apollo Client v4.

Reviewer: ```suggestion
Note that the ability to pass `addTypename: false` will not be supported with Apollo Client v4.
```

Saying "remove" makes it sound like we will remove that option from the config, but really its just that we can't support that option anymore. If someone uses this tool with AC3, it should continue to function just fine 🙂
